chain/jsonrpc-primitives/src/types/receipts.rs

```diff
@@ -0,0 +1,82 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ReceiptReference {
+    pub receipt_id: near_primitives::hash::CryptoHash,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct RpcReceiptRequest {
+    #[serde(flatten)]
+    pub receipt_reference: ReceiptReference,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct RpcReceiptResponse {
+    #[serde(flatten)]
+    pub receipt_view: Option<near_primitives::views::ReceiptView>,
```

Comment:
I think it is better to respond with a `ReceiptView`, and if there is no receipt found (`Option::None` variant) we can respond with `RpcReceiptError::UnknownReceipt`.